src/util/event.h

```diff
@@ -26,9 +26,34 @@ class Event {
     static bool start(const QString& tag) {
         return event(tag, Stat::EVENT_START);
     }
+
     static bool end(const QString& tag) {
         return event(tag, Stat::EVENT_END);
     }
+
+  private:
+    static bool event(const char*, Event::EventType) {
+        // this should not be uses to avoid unicode encoding and
+        // mamory alloc at every call. Use:
+        // static const QString tag(const char*);
+        return false;
+    }
+
+    static bool start(const char*) {
+        // this should not be uses to avoid unicode encoding and
+        // mamory alloc at every call. Use:
+        // static const QString tag(const char*);
```

Comment:
A comment should explain that those function solely exist as a placeholder to prevent the invocation of QString(const char*) at compile time and must not be removed!

Reply:
or use c++11 = delete